<?xml version="1.0" ?>
<launch>
  <node name="slam_logger_node" pkg="digiforest_drs" type="slam_logger_node" output="screen">
    <param name="lidar_topic"                 type="string" value="/vilens/point_cloud_transformed_processed" />

    <param name="odom_frame"                  type="string" value="odom_vilens" />
    <param name="odom_base_frame"             type="string" value="base_vilens" />

    <param name="output_file_type"            type="string" value="pcd" />
    <param name="reading_dist_threshold"      type="double" value="2.0" />
    <param name="platform_id"                 type="int"    value="1" />
  </node>
</launch>


